tests/lava/proc/graded/test_graded.py

```diff
@@ -204,16 +204,16 @@ def test_norm_vec_delay_out1(self):
         ch2 = (weights2 @ inp_data2) / 2**weight_exp
 
         # I'm using roll to account for the two step delay but hacky
```

Comment:
Do we not have a better explanation of the delay? If not suggest removing. This comment does not seem useful at all.